pkgs/tools/security/enpass/default.nix

```diff
@@ -0,0 +1,80 @@
+{stdenv, fetchurl, dpkg, patchelf, openssl, xorg
+, glib, mesa, libpulseaudio, zlib, dbus, fontconfig, freetype
+, gtk2, pango, atk, cairo, gdk_pixbuf, jasper, xkeyboardconfig
+, makeWrapper , makeDesktopItem, ...}:
+let
+  data = import ./data.nix;
+  uselibs = with xorg; [
+    openssl.out
```

Comment:
i believe `out` is the default. anyhow this can be removed completely because you only use it in makeLibraryPath (same for `glib`)